apps/mobile/src/features/ConfirmTx/components/ReviewAndConfirm/ReviewAndConfirmView.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { useTheme, View } from 'tamagui'
+import { Tabs, MaterialTabBar } from 'react-native-collapsible-tab-view'
+import { TransactionDetails } from '@safe-global/store/gateway/AUTO_GENERATED/transactions'
+import { ReviewHeader } from './ReviewHeader'
+import { ReviewFooter } from './ReviewFooter'
+import { DataTab } from './tabs/DataTab'
+import { JSONTab } from './tabs/JSONTab'
+import { Address } from '@/src/types/address'
+import { useTheme as useCurrentTheme } from '@/src/theme/hooks/useTheme'
+
+interface ReviewAndConfirmViewProps {
+  txDetails: TransactionDetails
+  signerAddress: Address
+}
+
+export function ReviewAndConfirmView({ txDetails, signerAddress }: ReviewAndConfirmViewProps) {
+  const { currentTheme } = useCurrentTheme()
+  const theme = useTheme()
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const renderTabBar = (props: any) => (
+    <MaterialTabBar
+      {...props}
+      indicatorStyle={{
+        backgroundColor: theme.color.get(),
+      }}
+      style={{ backgroundColor: currentTheme === 'light' ? theme.backgroundPaper.get() : theme.background.get() }}
+      labelStyle={{ color: theme.color.get(), fontSize: 16, fontWeight: '600' }}
+      activeColor={theme.color.get()}
+      inactiveColor={theme.colorSecondary.get()}
+      tabBarItemStyle={{
+        tabBarItemWidth: 100,
+        tabBarIndicatorWidth: 94,
+        tabBarLabelFontSize: 16,
+        tabBarLabelFontWeight: '700',
+        tabBarItemLeft: -10,
+      }}
+      width={200}
+    />
+  )
```

Comment:
While testing I was able to remove a few of the props without the appearance changing and also get rid of the any type. Can you check if it still looks the same for you as well?

```
const renderTabBar = (props: TabBarProps<string>) => (
    <MaterialTabBar
      {...props}
      indicatorStyle={{
        backgroundColor: theme.color.get(),
      }}
      labelStyle={{ fontSize: 16, fontWeight: '600' }}
      inactiveColor={theme.colorSecondary.get()}
      width={200}
    />
  )
```